zingolib/src/grpc_connector.rs

```diff
@@ -38,7 +38,7 @@ impl GrpcConnector {
         Self { uri }
     }
 
-    pub(crate) fn get_client(
+    pub fn get_client(
```

Comment:
does this change in public interface need a CHANGELOG entry of some sort or additional docs?